src/pages/batch-create-offers/BatchCreateOffers.tsx

```diff
@@ -0,0 +1,223 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { offers } from "@bosonprotocol/react-kit";
+import { Form, Formik } from "formik";
+import { useMemo, useRef, useState } from "react";
+import styled from "styled-components";
+
+import DetailTable, { Data } from "../../components/detail/DetailTable";
+import { FormField } from "../../components/form";
+import {
+  ContainerProductPage,
+  SectionTitle
+} from "../../components/product/Product.styles";
+import BosonButton from "../../components/ui/BosonButton";
+import Typography from "../../components/ui/Typography";
+import { useOffers } from "../../lib/utils/hooks/offers";
+import { useGetOfferMetadata } from "../../lib/utils/hooks/offers/useGetOfferMetadata";
+import { useCurrentSellers } from "../../lib/utils/hooks/useCurrentSellers";
+
+// see https://dev.to/atosh502/file-uploads-and-validation-with-react-and-formik-2mbk
+
+const FileUpload = ({ fileRef, name, onChange }: any) => {
+  return (
+    <div>
+      <label htmlFor="files">Choose files</label>{" "}
+      <input
+        ref={fileRef}
+        multiple={true}
+        type="file"
+        name={name}
+        onChange={onChange}
+      />
+    </div>
```

Comment:
maybe we can use the existing Upload component?

Reply:
I've tried but I find the Upload component is a bit too much oriented for images
![image](https://github.com/bosonprotocol/interface/assets/7184124/8177dceb-48bf-4c22-88ea-6183c51a0f7c)
Personally I have a preference for keeping the basic input...

Reply:
true, nvm